### PR TITLE
Add extra waypoints

### PR DESCRIPTION
**Pull request by drchrislewis**
need a full systems test

### CODE REVIEW

File: tool_path_planner/test/utest.cpp [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #define DISPLAY_LINES 1
 #define DISPLAY_NORMALS 0
 #define DISPLAY_DERIVATIVES 1
-#define POINT_SPACING 0.5
+#define POINT_SPACING 1.0
```
Comment (drchrislewis) [inline tool_path_planner/test/utest.cpp:22]: probably should revert this to 0.5

File: tool_path_planner/test/utest.cpp [hunk 2]
```diff
@@ -287,6 +329,7 @@ void runExtraRasterTest(tool_path_planner::PathGenerator& planner,
   color[1] = 0.9f;
   color[2] = 0.9f;
   viz.addPolyDataDisplay(mesh, color);
+  viz2.addPolyDataDisplay(mesh, color);
```
Comment (drchrislewis) [inline tool_path_planner/test/utest.cpp:344]: Added a second display because it was too confusing to have the old waypoints mixed with the new ones.

**Comment by colin-lewis-19** [comment]
This is working well @drchrislewis, provided it has halfway decent meshes. I think its good to go @DavidMerzJr

**Comment by DavidMerzJr** [comment]
I'll take care of the broken Clang-Format.  Thanks for the hard work guys!

**Pull request merged**